source/Coop/Mod/Managers/ClientCharacterCreatorManager.cs

```diff
@@ -10,30 +10,29 @@
 using TaleWorlds.CampaignSystem.CharacterCreationContent;
 using TaleWorlds.MountAndBlade.GauntletUI;
 using TaleWorlds.MountAndBlade.ViewModelCollection;
+using TaleWorlds.MountAndBlade;
+using TaleWorlds.Engine.Screens;
+using TaleWorlds.Engine;
+using TaleWorlds.Localization;
+using System.Collections.Generic;
+using Coop.Mod.Extentions;
+using Coop.Mod.Serializers;
+using SandBox;
 
 namespace Coop.Mod.Managers
 {
     public class HeroEventArgs : EventArgs
     {
-
-        public ObjectId HeroId { get; private set; }
-        public string PartyName { get; private set; }
-        public HeroEventArgs(string PartyName, ObjectId HeroId)
-        {
-            this.PartyName = PartyName;
-            this.HeroId = HeroId;
-        }
+        public PlayerHeroSerializer SerializedHero { get; private set; }
     }
-    public class ClientCharacterCreatorManager : StoryModeGameManager
+    public class ClientCharacterCreatorManager : SandBoxGameManager

```

Comment:
I've been curious about this - Is this just v1 - getting sandbox working? And we have plans down the line to get the actual campaign mode synced up?